istio/datadog_checks/istio/data/conf.yaml.example

```diff
@@ -1,52 +1,462 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
-instances:
-
-    ## @param send_histograms_buckets - boolean - required
-    ## Set send_histograms_buckets to true to send the histograms bucket from Istio.
```

Comment:
Not sure why this was required, the openmetrics template has this as `true` by default and optional.